zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/multitenancy/TenantAwareModifyProcessInstanceTest.java

```diff
@@ -72,49 +71,6 @@ public void shouldModifyInstanceForDefaultTenant() {
         .hasIntent(ProcessInstanceModificationIntent.MODIFIED);
   }
 
-  @Test
-  public void shouldRejectModifyInstanceForUnauthorizedTenant() {
```

Comment:
❌ This test also seems relevant to me still. I wouldn't want to delete this.

Reply:
Same problem with all the tenantAwareTest, the authorization is not enabled